.travis.yml

```diff
@@ -1,29 +1,35 @@
 language: ruby
+cache: bundler
+
 before_install:
   - gem install -v '1.17.3' bundler
-install: bundle _1.17.3_ install
-bundler_args: --jobs 3 --retry 3 --without benchmark
+bundler_args: --without benchmark
+
 os:
   - linux
   - osx
+
 rvm:
-  - 2.5.0
-  - 2.4.3
-  - 2.3.5
-  - 2.2.9
-  - 2.1.10
-  - 2.0.0
-  - 1.9.3
+  - 2.7
+  - 2.6
+  - 2.5
+  - 2.4
+  - 2.3
+  - 2.2
+  - 2.1
+  - 2.0
+  - 1.9
```

Comment:
The patch versions were mostly out of date, this will pick whatever latest patch version is available on Travis which should also speed up CI runs a bit since existing versions will be used.

I also added 2.6 and 2.7, let's see if that works, since they ship with a newer version of bundler, not 1.17.3.